commands/georadius.md

```diff
@@ -60,5 +60,5 @@ Both commands were introduced in Redis 3.2.10 and Redis 4.0.0 respectively.
 GEOADD Sicily 13.361389 38.115556 "Palermo" 15.087269 37.502669 "Catania"
 GEORADIUS Sicily 15 37 200 km WITHDIST
 GEORADIUS Sicily 15 37 200 km WITHCOORD
-GEORADIUS Sicily 15 37 200 km WITHDIST WITHCOORD
+GEORADIUS Sicily 15 37 200 km WITHCOORD WITHDIST
```

Comment:
I think it's better to change commands.json to list them in the order `[WITHDIST] [WITHHASH] [WITHCOORD]` since that's the order the information is returned when all (or two of them) are specified.